utils/ifquery

```diff
@@ -1,32 +1,34 @@
-#!/usr/bin/python
+#!/usr/bin/python3
```

Comment:
i'd argue we'd get [a few] complaints for a while from users wanting to run `master` on older systems;
i'm not fully aware what's the cutoff from 2 to 3 on all distros;
maybe we should let the default be `python` and let distros decide when `python` points to `python3`

Reply:
As far as I'm aware, this script is only being used on RPM-based distros (I wrote it specifically for use in RPMs).  And as far as I'm aware, I'm the only one actually building RPMs for `mstpd` at this time.
But if you're worried about it, I suppose I could have the `rpm.spec` file edit the `ifquery` file during the build.

Reply:
> As far as I'm aware, this script is only being used on RPM-based distros (I wrote it specifically for use in RPMs).

a bit hard to say whether you are the only one :)
the feedback we get is silence when things work;
i'm not against switching the shebang to `python3`;
we can do that;
i'll leave the preference up to you;